scripts/gulpfiles/package_tasks.js

```diff
@@ -396,9 +386,8 @@ const package = gulp.series(
         packageLua,
         packageDart,
         packagePHP,
-        packageLocales,
         packageMedia,
-        packageUMDBundle,
+        gulp.series(packageLocales, packageUMDBundle),
```

Comment:
Why do these have to be in series now?

Reply:
Because `blockly.min.js` includes the (wrapped) version of `msg/en.js`.